charts/selenium-grid/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this helm chart will be documented in this file.
 
+## :heavy_check_mark: 0.22.1
+
+### Changed
+- Update image tag to 4.14.0-20231010
```

Comment:
I was thinking about this again; this file cannot be changed before the release happens. Can you please revert this change and send it in a separate PR?